data/styles/clear/include/Icons.mapcss

```diff
@@ -2932,14 +2932,14 @@ node|z15-[amenity=sanitary_dump_station],
 area|z15-[amenity=sanitary_dump_station],
 node|z16-[amenity=charging_station],
 area|z16-[amenity=charging_station],
-node|z17-[amenity=bicycle_parking],
-area|z17-[amenity=bicycle_parking],
 node|z17-[amenity=motorcycle_parking],
 area|z17-[amenity=motorcycle_parking],
 node|z17-[amenity=car_wash],
 area|z17-[amenity=car_wash],
 node|z18-[amenity=parking],
 area|z18-[amenity=parking],
+node|z18-[amenity=bicycle_parking],
+area|z18-[amenity=bicycle_parking],
```

Comment:
did we want to demote bicycle parkings from z17 to z18? I don't remember it in the discussion - cyclists won't be happy :)

Reply:
it wasn't mentioned in the discussion, but in cities they can cause a lot of visual clutter. i'll leave it at z17 for now 👍